### PR TITLE
Make StorageClass PV reclaim policy configurable

### PR DESCRIPTION
**Pull request by vegichan**
The reclaimPolicy was hardcoded to `Retain`. Make it configurable in the values.yaml.

### CODE REVIEW

File: stable/storage-class/values.yaml [hunk 1]
```diff
@@ -15,6 +15,15 @@ storageClass:
   # Default is true for NuoDB.
   allowVolumeExpansion: true
 
+  # PersistentVolumes can have various reclaim policies, including "Retain", "Recycle", and "Delete".
```
Comment (NikTJ777) [inline stable/storage-class/values.yaml:18]: Do we want to add something to the README about reclaimPolicy?

Comment (vegichan) [reply stable/storage-class/values.yaml:18]: good catch, thanks. Will add

File: stable/storage-class/README.md [hunk 2]
```diff
@@ -42,6 +42,17 @@ cloud:
...
+#### storageClass.*
+
+The purpose of this section is to specify the storage class parameters. These parameters are applied to all storage classes created by this chart.
+
+The following tables list the configurable parameters for the `storageClass` option:
+
+| Parameter | Description | Default |
+| ----- | ----------- | ------ |
+| `allowVolumeExpansion` | Persistent Volumes can be configured to be expandable. This feature when set to true, allows the users to resize the volume by editing the corresponding PVC object. |`true`|
+| `reclaimPolicy` | PersistentVolumes can have various reclaim policies, including "Retain", "Recycle", and "Delete". For dynamically provisioned PersistentVolumes, the default reclaim policy is "Delete". This means that a dynamically provisioned volume is automatically deleted when a user deletes the corresponding PersistentVolumeClaim. |`Retain`|
```
Comment (NikTJ777) [inline stable/storage-class/README.md:54]: Is the default 'Retain' or 'Delete'?

Comment (vegichan) [reply stable/storage-class/README.md:54]: The kubernetes default is Delete. Our default is Retain

Comment (NikTJ777) [reply stable/storage-class/README.md:54]: The suggest a minor change to wording:
"For dynamically provisioned PersistentVolumes, the default kubernetes reclaim policy is "Delete".
NuoDB changes this default to "Retain".

Comment (vegichan) [reply stable/storage-class/README.md:54]: thanks!

**Pull request merged**